pom.xml

```diff
@@ -16,22 +16,22 @@
 -->
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
   <modelVersion>4.0.0</modelVersion>
-  <parent>
-    <groupId>dev.snowdrop</groupId>
-    <artifactId>snowdrop-dependencies</artifactId>
-    <version>2.5.12.Final</version>
-  </parent>
-
   <groupId>dev.snowdrop.example</groupId>
   <artifactId>rest-http</artifactId>
-  <version>2.5.12-3-SNAPSHOT</version>
-  <name>Spring Boot - HTTP Example</name>
-  <description>Spring Boot - HTTP Example</description>
+  <version>2.11.0-SNAPSHOT</version>
+  <name>Dekorate - Spring Boot - HTTP Example</name>
+  <description>Dekorate - Spring Boot - HTTP Example</description>
```

Comment:
I don't like to rename the examples adding `Dekorate`. Dekorate is just an utility to generate resources. For example, we also use `helm` in the snowdrop examples.